docs/models/basics.mdx

```diff
@@ -88,7 +88,8 @@ You can add other parameters or override this behavior with [passthrough](#passt
     "model": "gpt-3.5-turbo",
     "prompt": "Hey I'm {{user_name}}",
     "parameters": {
-      "temperature": 0.1
+      "temperature": 0.1,
+      "timeout": 2000
```

Comment:
i think we should add this to the parameter types for both `type: model` and `type: py-script` run configurations

this will also need to be changed to how passthrough parameters work. see `getPassthroughParams`

Reply:
also let's think through how to document this parameter